lib/resources.js

```diff
@@ -118,6 +129,9 @@ module.exports = function (app) {
         term: {
           uris: params.uri
         }
+      },
+      _source: {
+        excludes: EXCLUDE_FIELDS
```

Comment:
Tell ES to exclude fields that match `['uris', '*_packed', '*_sort', 'items.*_packed']`